apis/handlers/addprog_test.go

```diff
@@ -0,0 +1,104 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/l3af-project/l3afd/config"
+	"github.com/l3af-project/l3afd/kf"
+)
+
+const dummypayload string = `[
+	{
+	  "host_name" : "l3af-local-test",
+	  "iface" : "mockif0",
```

Comment:
nfconfig_test.go uses "fakeif0" and here and other files use "mockif0".
Can we pick one and be consistent?